.github/workflows/format.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           cache: 'npm'
       # Workflows are not allowed to edit workflows. As result, we need to prevent Prettier from formatting them.
       - name: Prevent workflows from being formatted
-        run: echo ".github" >> .prettierignore
+        run: echo ".github" >> .prettierignore && cat .prettierignore
```

Comment:
This line wasn't working correctly and was failing the build as the `.prettierignore` file didn't have a newline character at the end of the file. So instead of appending a new line item this was appending to the end of the last item

```
node_modules/
doc/api.github
```